### PR TITLE
Create rudimentary consumer and link to Kafka cluster

### PR DESCRIPTION
**Pull request by sebjshaw**
Created a mini consumer, this listens to the Kafka cluster and gets the data every second

### CODE REVIEW

File: deloton_test/deloton_kafka_consumer.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+import os
+from confluent_kafka import Consumer
+import json
+from dotenv import load_dotenv
+load_dotenv()
+
+
+topic = 'deloton'
```
Comment (AlexSkowronski2) [inline deloton_test/deloton_kafka_consumer.py:9]: NICE

File: deloton_test/deloton_kafka_consumer.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    # 'heartbeat.interval.ms': 1000,
+    # 'fetch.wait.max.ms': 6000,
+    'auto.offset.reset': 'latest',
+    'enable.auto.commit': 'false',
+    'max.poll.interval.ms': '86400000',
+    'topic.metadata.refresh.interval.ms': "-1",
+    # "client.id": 'id-002-005',
+})
+
+c.subscribe([topic])
```
Comment (AlexSkowronski2) [inline deloton_test/deloton_kafka_consumer.py:29]: chayzeeee

**Review by AlexSkowronski2 — COMMENTED** [review]
This is my review of your terrible code!!!

**Review by DomLaw82 — APPROVED** [review]
Perfect for streaming top quality data

**Pull request merged**